src/components/middle/MessageList.tsx

```diff
@@ -215,6 +215,38 @@ const MessageList: FC<OwnProps & StateProps> = ({
     memoFirstUnreadIdRef.current = firstUnreadId;
   }, [firstUnreadId]);
 
+  useEffect(() => {
```

Comment:
`captureEvents: onSwipe` is recommended for these purposes.

Reply:
In this PR, I propose adding functionality to handle back swipes specifically for Touch Bar swipes. The `onSwipe` handler in `captureEvents` is designed for touch and mouse events and does not detect `wheel` events generated by the MacBook's Touch Bar. The Touch Bar's unique `wheel` events have different characteristics and thresholds, which are not compatible with the `onSwipe` logic that's tailored for touch and mouse gestures.

Reply:
I see. I think that still needs to be handled within `captureEvent` module which already have this behavior [for Media Viewer](https://github.com/Ajaxy/telegram-tt/blob/master/src/components/mediaViewer/MediaViewerSlides.tsx#L425) (`onWheelDrag`).